package.json

```diff
@@ -2,34 +2,37 @@
   "name": "react-lifecycles-compat",
   "version": "1.1.0",
   "description": "Backwards compatibility polyfill for React class components",
-  "main": "index.js",
+  "main": "react-lifecycles-compat.cjs.js",
+  "module": "react-lifecycles-compat.es.js",
   "license": "MIT",
   "repository": "reactjs/react-lifecycles-compat",
   "scripts": {
+    "pretest": "npm run build",
```

Comment:
this script got moved accidently when rebasing

Reply:
Tiny nit: I prefer alpha-sorting these :)